pkg/imageservice/endpoints.go

```diff
@@ -66,16 +88,42 @@ func MakeDeleteImageEndpoint(svc resizer.ImageService) endpoint.Endpoint {
 	}
 }
 
+func MakeScheduleResizeJobEndpoint(svc resizer.ImageService) endpoint.Endpoint {
+	return func(ctx context.Context, request interface{}) (interface{}, error) {
+		req := request.(createResizeRequest)
+		job, err := svc.ScheduleResizeJob(ctx, req.ImgId, req.Width, req.Height)
+		if err != nil {
+			return createResizeResponse{err.Error(), resizer.StatusFailed, time.Time{}}, nil
+		}
+		return createResizeResponse{"", job.Status, job.CreatedAt}, nil
+	}
+}
+
+func MakeGetResizeJobEndpoint(svc resizer.ImageService) endpoint.Endpoint {
+	return func(ctx context.Context, request interface{}) (interface{}, error) {
+		req := request.(getResizeJobRequest)
+		job, err := svc.GetResizeJob(ctx, req.JobId)
+		if err != nil {
+			return getResizeJobResponse{nil, resizer.ErrNoImage}, nil
```

Comment:
Where do you dispatch the error? Why do you not return error as the second value?